### ISSUE TITLE
There are really two MetalLB VIPs?

### ISSUE THREAD

**Issue by flozanorht** [openstack-k8s-operators/docs#82]
In https://github.com/openstack-k8s-operators/docs/blob/main/images/network_diagram.jpg there are two VIPs, both for internal API traffic (which I assume means OpenStack service APIs) but in the sample configs from the devpreview2 docs I see only one such VIP. All OpenStack services which include MetalLB settings share the same IP. 

Maybe one of those IPs refer to cell-specific instances of RabbitMQ? This is the only different VIP I noticed from the devprevire2 docs.

**Comment by stuggi** [comment]
yes, you need dedicated IPs for rabbitmq instances as its IP based. they use the same port and therefore can not share the same ip. if it is an issue in the devpeview2 docs it should be fixed there. this repo is not for those docs.

**Comment by flozanorht** [comment]
Is there a way you can differentiate (maybe with some text around the figure) the network path for openstack services (nova, neutron, etc) and for its infrastructure services (rabbitmq, mariadb)? Or maybe indicate what's per control plane and what's per cell?

I mention the devpreview2 docs because it alone (and this git repo docs alone) are not sufficient for me to get a clear picture of OSP18 networking requirements... some information I see only in one of them, and some information seems to contradict each other... and it seems sometimes different names are given to the same thing.

I agree some issues must be solved by the product docs team on the devpreview2 docs and I created some request there too.
